### DISCUSSION TITLE
How to exclude transactions from monthly spending?

### DISCUSSION THREAD

**Discussion by dekiesel** [firefly-iii/firefly-iii#8724] [Firefly III] [answered]
Hi,

Is there a way to exclude a transaction (two, in my case) from monthly spending?

Situation:

I have a small investment fund for my kid. The grandparents transfered 1000eu into my account, which I then put into my kids fund. 

The issue is that a) My yearly income is too high (only on paper ;))  and my monthly expense shows 1000eu too much 

It's not my money though so I don't think I can/should turn the fund into an asset account.

Any ideas?

**Comment by JC5** [comment, marked as answer]
You can exclude the asset account from your reports, and remove the asset account from your net worth, but otherwise the income will remain visible.

**Reply by dekiesel** [reply]
That's a bummer, but I guess it makes sense. Thank you for your answer!
